src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-11-1-upgrade-notes.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Upgrading Nuxeo Web UI from LTS 2019 to 11.1
+description: Upgrade notes from Nuxeo Web UI LTS 2019 (10.10) to 11.1
+review:
+    comment: ''
+    date: '2020-06-24'
+    status: ok
+toc: true
+labels:
+tree_item_index: 999
+---
+
+{{! multiexcerpt name='upgrade-notes'}}
+
+{{! multiexcerpt name='matching-notes'}}
+{{#> callout type='info' heading='Release Notes'}}
+This page mentions how to do a technical upgrade. Have a look at the [release notes]({{page page='web-ui-11-1-release-notes'}}) to see what's new.
+{{/callout}}
+{{! /multiexcerpt}}
+
+## Upgrading from Web UI LTS 2019 to Web UI 11.1
+
+### Polymer 3 Migration
+
+#### High Level Overview
+Migrating to Polymer 3 is fairly transparent. Quoting the <a href="https://polymer-library.polymer-project.org/3.0/docs/about_30#api-changes" target="_blank">Polymer dev team</a>:
+
+> The Polymer 3.0 API is almost 100% backward compatible with Polymer 2.x—the only changes are removing APIs related to HTML Imports (such as importHref), and converting Polymer's API to be module-based rather than globals-based.
+
+#### Studio Projects
+Studio configuration is made to be compatible and does not need to be migrated. If you did add customization in your project, see below for further details. Otherwise you can consider your project safe to use.
+
+#### HTML Imports
+Nuxeo added a compatibility layer to keep HTML imports working. There is no need to update HTML imports done in your code.
+
+#### ES Modules
+Polymer 3 is enforcing the use of modules, and requires to import specific features rather than using wildcards. Some options that were available under the global `Polymer` namespace will not work by default anymore: e.g. there is no `Polymer.dom.*`, you are supposed to do this instead:
```

Comment:
```suggestion
Because Polymer 3 relies on ES Modules, it no longer loads HTML files through HTML imports, which would set Polymer objects into the global `Polymer` namespace. Therefore, methods and objects that were available under the `Polymer` namespace will not work by default anymore: e.g. there is no `Polymer.dom.*`, you are supposed to do this instead:
```